setup.py

```diff
@@ -1,24 +1,22 @@
-#!/usr/bin/env python
 from itertools import chain
```

Comment:
can you explain the reason for the changes in this file? I don't think people should run the tests through the setup.py file so we can probably remove the test requires stuff.